index.js

```diff
@@ -46,30 +46,39 @@ cs.get.rgb = function (string) {
 		return null;
 	}
 
-	var abbr = /^#([a-fA-F0-9]{3})$/;
-	var hex = /^#([a-fA-F0-9]{6})$/;
+	var abbr = /^#([a-f0-9]{3})([a-f0-9])?$/i;
```

Comment:
In fact, could you add a test to make sure a 7 digit hex string _doesn't_ match? Just to be sure.